src/Sarif.Converters/FortifyConverter.cs

```diff
@@ -102,16 +93,10 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
                         Text = runDescription
                     }
                 },
-                Tool = tool
+                Tool = new Tool {  Name = "Fortify" }
```

Comment:
>Fortify [](start = 43, length = 7)

Right, here's a case where you can't use the format for the tool name, because there are two of them. Since you can't do it always, I think you should never do it; it's more or less an accident if the format name happens to match the tool name. Besides, the tool's preferred name might not even be a valid identifier (_e.g._, "Clang Analyzer").